crypto/bn/bn_rsa_fips186_4.c

```diff
@@ -49,35 +49,33 @@ const BIGNUM ossl_bn_inv_sqrt_2 = {
 };
 
 /*
- * Refer to FIPS 186-5 Table B.2 for minimum rounds of Miller Rabin
- * required for generation of RSA aux primes (p1, p2, q1 and q2)
- * using an error probability of 2^–100.
+ * Refer to FIPS 186-5 Table B.1 for minimum rounds of Miller Rabin
+ * required for generation of RSA aux primes (p1, p2, q1 and q2).
  */
 static int bn_rsa_fips186_5_aux_prime_MR_rounds(int nbits)
 {
     if (nbits >= 4096)
-        return 22;
+        return 44;
     if (nbits >= 3072)
-        return 27;
+        return 41;
     if (nbits >= 2048)
-        return 32;
-    return 0;
+        return 38;
+    return 0; /* Error */
 }
 
 /*
- * Refer to FIPS 186-5 Table B.2 for minimum rounds of Miller Rabin
+ * Refer to FIPS 186-5 Table B.1 for minimum rounds of Miller Rabin
  * required for generation of RSA primes (p and q)
- * using an error probability of 2^–100.
  */
 static int bn_rsa_fips186_5_prime_MR_rounds(int nbits)
 {
     if (nbits >= 4096)
```

Comment:
Just delete this branch, it's the same as the >= 3072 one below.

Reply:
Done